server/models/squad.ts

```diff
@@ -17,7 +16,7 @@ export const squadSchema = new Schema(
 );
 
 // Before saving the squad, hash the password
-squadSchema.pre('save', function (next) {
+squadSchema.pre('save', next => {
```

Comment:
this can break the code, there is an issue with using arrow function with mongoose